### PR TITLE
moved locale picker inside sign-in + sign-up session

### PR DESCRIPTION
**Pull request by sudheshna-donthineni**

### CODE REVIEW

File: src/pages/Home.vue [hunk 1]
```diff
@@ -18,7 +18,6 @@
 
   <div v-else>
     <div v-if="!oldFlow">
-      <LocalePicker :options="getLocale" />
```
Comment (sudheshna-donthineni) [inline src/pages/Home.vue:21]: removed this from `Home` into `NewSignIn`

File: src/pages/Home.vue [hunk 2]
```diff
@@ -322,7 +314,9 @@ export default {
     isTypeSignIn() {
       return (
         (this.sessionData && this.sessionData.type == "sign-in") ||
-        (this.type == "sign-in" && this.oldFlow == false) ||
+        (!this.sessionData &&
```
Comment (sudheshna-donthineni) [inline src/pages/Home.vue:317]: makes sure that sessionData does not exist before taking the default value.

File: src/pages/NewSignin.vue [hunk 3]
```diff
@@ -334,17 +343,20 @@ export default {
       );
 
       if (!isUserValid.isUserIdValid) {
-        this.invalidLoginMessage = this.invalidLoginMessageTranslations["ID"][
```
Comment (sudheshna-donthineni) [inline src/pages/NewSignin.vue:337]: formatting. Don't know why it's happening.

File: src/pages/NewSignup.vue [hunk 4]
```diff
@@ -50,13 +50,17 @@
...
+      v-if="$store.state.sessionData.type == 'sign-in'"
+      class="mt-[30px] flex w-48 mx-auto justify-between items-center"
+    >
       <hr class="w-20 text-grey" />
       <p class="text-grey font-roboto text-sm opacity-40">or</p>
       <hr class="w-20 text-grey" />
     </div>
     <!-- signup button -->
     <button
+      v-if="$store.state.sessionData.type == 'sign-in'"
```
Comment (sudheshna-donthineni) [inline src/pages/NewSignup.vue:63]: Sign in allowed in sign-up only when required.

File: src/pages/NewSignup.vue [hunk 5]
```diff
@@ -50,13 +50,17 @@
...
-    <div class="mt-[30px] flex w-48 mx-auto justify-between items-center">
+    <div
+      v-if="$store.state.sessionData.type == 'sign-in'"
+      class="mt-[30px] flex w-48 mx-auto justify-between items-center"
+    >
       <hr class="w-20 text-grey" />
       <p class="text-grey font-roboto text-sm opacity-40">or</p>
       <hr class="w-20 text-grey" />
     </div>
     <!-- signup button -->
```
Comment (suryabulusu) [inline src/pages/NewSignup.vue]: why is this comment between "or" and "sign-in" redirect button

**Review by suryabulusu — APPROVED** [review]

**Pull request merged**